src/IO/read_input.jl

```diff
@@ -142,8 +142,9 @@ if !ac_type_fixed
 end
 
 maxpax = readmis("max_payload_in_pax_equivalent") #This represents the maximum aircraft payload in equivalent number of pax
+                                                #Part of this payload may be carried in the aircraft's cargo hold
```

Comment:
do you mean to clarify that maxpax includes tonnage beyond passenger-associated weight?

If so, consider rephrasing something like, "this may exceed the seatable capacity to account for belly cargo."

tangential question, is this extra mass assumed to distribute uniformly along the cabin? if so, seems worth noting in the docs (if not already done).

Reply:
I didn't make changes to the way that the maximum payload is handled by 'fuseW()'. My understanding is that it is indeed uniformly distributed.